packages/cloud/deployment_details/deployment_details.tsx

```diff
@@ -0,0 +1,132 @@
+ * Side Public License, v 1.
+ */
+import React from 'react';
+
+import {
+  EuiForm,
+  EuiFormRow,
+  EuiFieldText,
+  EuiCopy,
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiButtonEmpty,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useDeploymentDetails } from './services';
+
+export const DeploymentDetails = ({ closeModal }: { closeModal?: () => void }) => {
+  const { cloudId, elasticsearchUrl, managementUrl, learnMoreUrl, navigateToUrl } =
+    useDeploymentDetails();
+  const isInsideModal = !!closeModal;
+
+  if (!cloudId) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiForm component="div">
+        {/* Elastic endpoint */}
+        {elasticsearchUrl && (
+          <EuiFormRow
+            label={i18n.translate('cloud.deploymentDetails.elasticEndpointLabel', {
+              defaultMessage: 'Elastic endpoint',
+            })}
+            fullWidth
+          >
+            <EuiFlexGroup gutterSize="s">
+              <EuiFlexItem>
+                <EuiFieldText
+                  value={elasticsearchUrl}
+                  fullWidth
+                  disabled
+                  data-test-subj="deploymentDetailsEsEndpoint"
+                />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiCopy textToCopy={elasticsearchUrl}>
+                  {(copy) => (
+                    <EuiButtonIcon
+                      onClick={copy}
+                      iconType="copyClipboard"
+                      display="base"
+                      size="m"
+                    />
+                  )}
+                </EuiCopy>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFormRow>
+        )}
+
+        {/* Cloud ID */}
+        <EuiFormRow
+          label={i18n.translate('cloud.deploymentDetails.cloudIDLabel', {
+            defaultMessage: 'Cloud ID',
+          })}
+          fullWidth
+        >
+          <EuiFlexGroup gutterSize="s">
+            <EuiFlexItem>
+              <EuiFieldText
+                value={cloudId}
+                fullWidth
+                disabled
+                data-test-subj="deploymentDetailsCloudID"
+              />
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiCopy textToCopy={cloudId}>
+                {(copy) => (
+                  <EuiButtonIcon onClick={copy} iconType="copyClipboard" display="base" size="m" />
+                )}
+              </EuiCopy>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFormRow>
+
+        <EuiSpacer size="m" />
+
+        {managementUrl && (
+          <EuiFlexGroup gutterSize="m" justifyContent="spaceBetween" alignItems="center">
+            <EuiFlexItem grow={false}>
+              {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
+              <EuiButtonEmpty
+                href={managementUrl}
+                onClick={(e: any) => {
```

Comment:
Could be something like

```suggestion
                onClick={(e: React.MouseEvent) => {
```

Reply:
Yeah that's better 👍